samples/transform_feedback_separated.html

```diff
@@ -0,0 +1,237 @@
+
+        var programFeedback = createProgram(gl, getShaderSource('vs-feedback'), getShaderSource('fs-feedback'));
+
+        var mvpLocation = gl.getUniformLocation(programTransform, 'MVP');
+
+        var PROGRAM_TRANSFORM = 0;
+        var PROGRAM_FEEDBACK = 1;
+        var programs = [programTransform, programFeedback];
+
+        // -- Init Buffer
+
+        var VERTEX_COUNT = 6;
+        var vertices = new Float32Array([
+            -1.0, -1.0, 0.0, 1.0,
+             1.0, -1.0, 0.0, 1.0,
+             1.0, 1.0, 0.0, 1.0,
+             1.0, 1.0, 0.0, 1.0,
+            -1.0, 1.0, 0.0, 1.0,
+            -1.0, -1.0, 0.0, 1.0
+        ]);
+
+        var BufferType = {
+            VERTEX: 0,
+            POSITION: 1,
+            COLOR: 2,
+            MAX: 3
+        };
+
+        var buffers = new Array(BufferType.MAX);
+        for (var i = 0; i < BufferType.MAX; ++i) {
+            buffers[i] = gl.createBuffer();
+        }
+
+        // Transform buffer
+        gl.bindBuffer(gl.ARRAY_BUFFER, buffers[BufferType.VERTEX]);
+        gl.bufferData(gl.ARRAY_BUFFER, vertices, gl.STATIC_DRAW);
+        gl.bindBuffer(gl.ARRAY_BUFFER, null);
+
+        // Feedback empty buffers
+        gl.bindBuffer(gl.ARRAY_BUFFER, buffers[BufferType.POSITION]);
+        gl.bufferData(gl.ARRAY_BUFFER, vertices.length * Float32Array.BYTES_PER_ELEMENT, gl.STATIC_DRAW);
+        gl.bindBuffer(gl.ARRAY_BUFFER, null);
+
+        gl.bindBuffer(gl.ARRAY_BUFFER, buffers[BufferType.COLOR]);
+        gl.bufferData(gl.ARRAY_BUFFER, vertices.length * Float32Array.BYTES_PER_ELEMENT, gl.STATIC_DRAW);
+        gl.bindBuffer(gl.ARRAY_BUFFER, null);
+
+        // -- Init Vertex Array
+        var vertexArrays = [gl.createVertexArray(), gl.createVertexArray()];
+        gl.bindVertexArray(vertexArrays[PROGRAM_TRANSFORM]);
+
+        var vertexPosLocation = 0; // set with GLSL layout qualifier
+        gl.bindBuffer(gl.ARRAY_BUFFER, buffers[BufferType.VERTEX]);
+        gl.vertexAttribPointer(vertexPosLocation, 4, gl.FLOAT, false, 0, 0);
+        gl.bindBuffer(gl.ARRAY_BUFFER, null);
+        gl.enableVertexAttribArray(vertexPosLocation);
+
+        gl.bindVertexArray(null);
+
+        gl.bindVertexArray(vertexArrays[PROGRAM_FEEDBACK]);
+
+        var vertexPosLocationFeedback = 0; // set with GLSL layout qualifier
+        gl.bindBuffer(gl.ARRAY_BUFFER, buffers[BufferType.POSITION]);
+        gl.vertexAttribPointer(vertexPosLocationFeedback, 4, gl.FLOAT, false, 0, 0);
+        gl.enableVertexAttribArray(vertexPosLocationFeedback);
+
+        var vertexColorLocationFeedback = 3; // set with GLSL layout qualifier
+        gl.bindBuffer(gl.ARRAY_BUFFER, buffers[BufferType.COLOR]);
+        gl.vertexAttribPointer(vertexColorLocationFeedback, 4, gl.FLOAT, false, 0, 0);
+        gl.enableVertexAttribArray(vertexColorLocationFeedback);
+
+        gl.bindBuffer(gl.ARRAY_BUFFER, null);
+        gl.bindVertexArray(null);
+
+        // -- Render
+
+        render();
```

Comment:
Remove the `render` function (it is not needed and disrupts the flow) and include the code in this function.